janus_core/calculations/single_point.py

```diff
@@ -216,10 +216,16 @@ def _get_potential_energy(self) -> MaybeList[float]:
         MaybeList[float]
             Potential energy of structure(s).
         """
+        tag = f"{self.architecture}_energy"
         if isinstance(self.struct, list):
-            return [struct.get_potential_energy() for struct in self.struct]
+            energies = [struct.get_potential_energy() for struct in self.struct]
+            for energy, struct in zip(energies, self.struct):
+                struct.info[tag] = energy
```

Comment:
Really finicky thing and purely stylistic, but I think that it's best to keep things in same positions as much as possible through the code where possible.
```suggestion
            for struct, energy  in zip(self.struct, energies):
                struct.info[tag] = energy
```